incubator/elasticsearch/README.md

```diff
@@ -189,11 +202,3 @@ Create cluster with Storage class `ssd` on Kubernetes 1.5+
 ```
 $ helm install incubator/elasticsearch --name my-release --set data.storageClass=ssd,data.storage=100Gi
 ```
-
```

Comment:
I am guessing this removal was unintentional?

Reply:
That is correct, let me removed for cleanliness.